web/src/components/Uploading.js

```diff
@@ -65,11 +65,13 @@ render() {
             </div>
             <div className="yesOrNo">
               <h5>Would you like to have an image marker to render your AR models?</h5>
+              {/* CR * dropbown */}
               <select  id="dropbown" name="anchor" onChange={handleChange} value={anchor}>
                 <option value="yes">Yes</option>
                 <option value="no">No</option>
               </select>
             </div>
+              {/* CR * ternary */}
```

Comment:
Nested ternaries in JSX are already quite challenging to read--and the formatting of these files makes it harder! Just for good measure: get prettier in here!! 〰️